README.md

```diff
@@ -1373,6 +1377,26 @@ ANSWERS.addComponent('Facets', {
 });
 ```
 
+### Transforming Facets
+
+The `transformFacets` option of the Facets component allows facets data to be fully customized. The function takes in and returns an array of the answers-core DisplayableFacet which is described [here](https://github.com/yext/answers-core/blob/master/docs/answers-core.displayablefacet.md). The function also has access to the Facets config as the second parameter.
+
+Here's an example of using this option to customize a boolean facet:
+
+```js
+transformFacets: facets => {
+  facets.forEach(facet => {
```

Comment:
Do we want to be changing the input `facets` object? I think we would want them to construct a new facets object, right?

Reply:
ya, you're right. i'll update

Reply:
I updated the example to use JSON. Do you think we might need something more robust such as lodash? JSON might not be sufficient if the facet has a date in it

Reply:
Just updated to lodash

Reply:
spoke to Tom offline. We decided to create a deep copy within the facets component so that someone can safely modify the facets object passed into this function

Reply:
Sounds good! I would think we want to give them example code without side effects too, right?

Reply:
While the example modifies the param object, there aren't any side effects within the facets component since that object was created as a deep clone. I tested this out and confirmed that the internal state is not affected when the facets object is modified. We could give them an example with a lodash deep clone, but that seems unnecessary since that would mean the facets object would be deep cloned twice which just means extra computation for no additional benefit

Reply:
The example now no longer has any side effects